test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/FormTagHelperTest.cs

```diff
@@ -114,8 +114,8 @@ public class FormTagHelperTest
             var viewContext = CreateViewContext();
             var expectedAttribute = new TagHelperAttribute("method", method.ToString().ToLowerInvariant());
             var context = new TagHelperContext(
-                allAttributes: new ReadOnlyTagHelperAttributeList<IReadOnlyTagHelperAttribute>(
-                    new ReadOnlyTagHelperAttributeList<IReadOnlyTagHelperAttribute>(new List<IReadOnlyTagHelperAttribute> { expectedAttribute })),
+                allAttributes: new TagHelperAttributeList(
+                    new TagHelperAttributeList(new List<TagHelperAttribute> { expectedAttribute })),
```

Comment:
Couldn't this be just `new TagHelperAttributeList(new List<TagHelperAttribute> { expectedAttribute })`?